production/helm/loki/templates/ingester/statefulset-ingester-zone-a.yaml

```diff
@@ -41,7 +41,7 @@ spec:
       name: ingester-zone-a
       rollout-group: ingester
   updateStrategy:
-    type: OnDelete
+    type: {{ .Values.ingester.updateStrategy }}
```

Comment:
ditto: could you please make the whole `updateStrategy` configurable otherwise users won't be able to set the rollingUpdate